metalog.js

```diff
@@ -44,9 +44,19 @@ const logTypes = types => {
   return flags;
 };
 
-const normalizeStack = stack => stack.replace(/\s+at\s+/g, '\n\t');
+const normalizeStack = stack => (!stack.includes(' at ') ? stack :
```

Comment:
IMHO, it should be better to save `indexOf(' at ')` into a variable, use it here instead of `includes` and later, when you need an actual index.